.github/workflows/regression-report.yml

```diff
@@ -73,30 +117,44 @@ jobs:
               content: 'rocket'
             })
 
-      - name: Checkout repository
-        uses: actions/checkout@v4
+      - name: Restore Master Checkstyle Checkout
+        uses: actions/cache@v4
+        with:
+          path: .ci-temp/checkstyle
+          key: "${{ env.CACHE_KEY_MASTER }}"
+
+      - name: Restore PR Checkout
+        uses: actions/cache@v4
+        with:
+          path: .
+          key: "${{ env.CACHE_KEY_PR }}"
 
       - name: Parse comment
         id: parse
         env:
           COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
-            sed -E 's/.*(Github|GitHub), generate report for //g')
-          echo "CONFIG_BUNDLE_PATH=$CONFIG_BUNDLE_PATH"
-          ./.ci/append-to-github-output.sh "config_bundle_path" "$CONFIG_BUNDLE_PATH"
+          pattern1='^Git[Hh]ub, generate report for configs in PR description$'
+          pattern2='^Git[Hh]ub, generate report for'
+          pattern3='^Git[Hh]ub, generate report by config from'
```

Comment:
PATTERN_FOR_DESCRIPTION
PATTERN_FOR
PATTERN_BY_CONFIG_FROM

Reply:
done

Reply:
can we resolve this?